apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.usecase.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable, Scope } from '@nestjs/common';
+import { EnvironmentEntity, EnvironmentRepository } from '@novu/dal';
+import { GetMyEnvironmentsCommand } from './get-my-environments.command';
+
+@Injectable({
+  scope: Scope.REQUEST,
```

Comment:
A question, is it needed to set scope here?

Reply:
Looks like not, but it's not making a harm either I think.